src/pybind/mgr/dashboard_v2/controllers/rbd_mirroring.py

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import json
+import re
+
+from functools import partial
+
+import rbd
+
+from dashboard_v2.controllers.dashboard import Dashboard
+from ..tools import ApiController, RESTController, ViewCache
+from .. import logger
+
+
+@ApiController('rbdmirror')
+class RbdMirror(RESTController):
+
+    def __init__(self):
+        self.pool_data = {}
+
+    def list(self):
+        _, data = self._get_daemons_and_pools()
+        content_data = self._get_content_data()
```

Comment:
could  you replace this to: `status, content_data = self._get_content_data()`

Reply:
Yes, sure, will do.